tools/deviceGen/input/device_features.json

```diff
@@ -1,2354 +1,3608 @@
-        },
-        {
-            "groupComment": [
-                "[REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance"
+            "comment": ["widely supported but has performance overhead, so remains an optional feature to enable"]
+        },
+        {
+            "comment": [
+                "Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance"
+            ],
+            "entries": [
+                {
+                    "type": "bool",
+                    "name": "fullDrawIndexUint32",
+                    "value": true,
+                    "expose": "DISABLE"
+                },
+                {
+                    "type": "bool",
+                    "name": "imageCubeArray",
+                    "value": true,
+                    "expose": "DISABLE"
+                },
+                {
+                    "type": "bool",
+                    "name": "independentBlend",
+                    "value": true,
+                    "expose": "DISABLE"
+                }
             ]
         },
         {
-            "type": "bool",
-            "name": "fullDrawIndexUint32",
-            "value": true,
-            "expose": false
-        },
-        {
-            "type": "bool",
-            "name": "imageCubeArray",
-            "value": true,
-            "expose": false
-        },
-        {
-            "type": "bool",
-            "name": "independentBlend",
-            "value": true,
-            "expose": false
-        },
-        {
-            "groupComment": [
+            "comment": [
                 "I have no clue if these cause overheads from simply being enabled"
+            ],
+            "entries": [
+                {
+                    "type": "bool",
+                    "name": "geometryShader",
+                    "value": false
+                },
+                {
+                    "type": "bool",
+                    "name": "tessellationShader",
+                    "value": false
+                }
             ]
         },
-        {
-            "type": "bool",
-            "name": "geometryShader",
-            "value": false
-        },
-        {
-            "type": "bool",
-            "name": "tessellationShader",
-            "value": false
-        },
         {
             "type": "bool",
             "name": "sampleRateShading",
             "value": true,
-            "expose": false,
-            "comment": "[REQUIRE] ROADMAP 2022 and good device support"
+            "expose": "DISABLE",
+            "comment": ["ROADMAP 2022 and good device support"]
         },
         {
             "type": "bool",
             "name": "dualSrcBlend",
             "value": true,
-            "expose": false,
-            "comment": "[REQUIRE] good device support"
+            "expose": "DISABLE",
```

Comment:
can you use `HIDE` or `REQUIRE` instead of `DISABLE`

DISABLE makes it seem like we're disabling the feature, when in fact `value=true` and we're requiring it

Reply:
probably REQUIRE and DISABLE, because:
- the ETC texture stuff you currently DISABLE
- but the required features you also DISABLE
which is mightily confusing